### ISSUE TITLE
packet loss is not shown on graphs

### ISSUE THREAD

**Issue by aspiers** [20c/vaping#13]
My graphs consistently show `loss 0 / 0` but concurrent `ping` and `mtr` invocations on the same machine show 10-15% packet loss.  Is this a bug or is my config wrong?  My `plugins` section looks like this:

```
plugins:
  - name: std_fping
    type: fping
    count: 10
    interval: 3s
    output:
      - vodka

  - name: vodka
    type: vodka
    logging:
      version: 1
      formatters:
        simple:
          format: '%(asctime)s - %(name)s - %(levelname)s: %(message)s'
      handlers:
        console:
          class: logging.StreamHandler
          level: DEBUG
          formatter: simple
          stream: ext://sys.stdout
      loggers:
        vodka:
          level: DEBUG
          handlers:
            - console

    data:
      - type: fping
        handlers:
          - type: index
            index: host
          - type: store
            container: list
            limit: 500

    apps:
      graphsrv:
        tmpl_engine: jinja2
        enabled: true
        graphs:
          multitarget:
            id_field: host
            type: multitarget
            plot_y: avg
            format_y: ms

          smokestack:
            id_field: host
            type: smokestack
            plot_y: avg

    plugins:

      - name: http
        type: flask
        host: 0.0.0.0
        port: 7021
        debug: true
        static_url_path: /static
        server: self
        async: thread
        routes:
          /targets : graphsrv->targets
          /graph_data :
            methods:
              - POST
              - GET
            target: graphsrv->graph_data
          /graph : graphsrv->graph_view
          /overview_read_file : graphsrv->overview_read_file
          /: graphsrv->overview_view
```

**Comment by vegu** [comment]
This is an issue with graphsrv, made a corresponding ticket here: https://github.com/20c/graphsrv/issues/13, thanks for the report

**Comment by vegu** [comment]
fixed in graphsrv 1.1.1

**Comment by aspiers** [comment]
Thanks! Have upgraded and restarted and now looking forward to seeing some non-zero counters :-)
